recipes/gnu-getopt/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+export CFLAGS="-I${PREFIX}/include"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib -L${PREFIX}/include -lintl"
```

Comment:
Please make sure the original environment variable's value is included in the assignment.

Reply:
Sure.